### DISCUSSION TITLE
Question about Aiohttp codes

### DISCUSSION THREAD

**Discussion by New-dev0** [aio-libs/aiohttp#6775] [General]
Can I know the reason of import as ?
are the names already same ?
Does doing that, will increases size of aiohttp and nothing  more ?
https://github.com/aio-libs/aiohttp/blob/b9c29a79d4498d8eebb0fbd368d5e4f24747d164/aiohttp/web.py#L25-L45

**Comment by Dreamsorcerer** [comment]
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport
